CMakeLists.txt

```diff
@@ -4,12 +4,12 @@ project(Kuzu VERSION 0.1.0 LANGUAGES CXX C)
 
 find_package(Threads REQUIRED)
 
-set(CMAKE_FIND_PACKAGE_RESOLVE_SYMLINKS TRUE)
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_C_VISIBILITY_PRESET hidden)
+set(CMAKE_FIND_PACKAGE_RESOLVE_SYMLINKS TRUE)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Are changes at L11 and L12 related to LTO?

Reply:
No, I just sorted these `set` statements lexicographically.